src/assets/CollectiblesController.ts

```diff
@@ -559,24 +562,19 @@ export class CollectiblesController extends BaseController<
       );
 
       if (existingEntry) {
-        const differentMetadata = compareCollectiblesMetadata(
-          collectibleMetadata,
-          existingEntry,
-        );
-        if (differentMetadata) {
-          // TODO: Switch to indexToUpdate
-          const indexToRemove = collectibles.findIndex(
-            (collectible) =>
-              collectible.address.toLowerCase() === address.toLowerCase() &&
-              collectible.tokenId === tokenId,
-          );
-          /* istanbul ignore next */
-          if (indexToRemove !== -1) {
-            collectibles.splice(indexToRemove, 1);
-          }
-        } else {
+        if (isCollectibleMetadataEqual(collectibleMetadata, existingEntry)) {
           return collectibles;
         }
+        // TODO: Switch to indexToUpdate
```

Comment:
This comment has been here for a while now. Are we going to do this change in the future or is it ok to remove it?

Reply:
addressed here: https://github.com/MetaMask/controllers/pull/680/commits/21847f8049686b1189a5dc9da36b3d69a31d2bdf